src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -221,8 +237,13 @@ BLEManagerImpl BLEManagerImpl::sInstance;
 CHIP_ERROR BLEManagerImpl::_Init()
 {
     CHIP_ERROR err;
+#ifndef SI917
     rsi_semaphore_create(&sl_rs_ble_init_sem, 0);
     rsi_semaphore_create(&sl_ble_event_sem, 0);
+#else
+    sl_rs_ble_init_sem = osSemaphoreNew(1, 0, NULL); // Matter
```

Comment:
the comment doesn't mean anything. you can remove it.

I think the `rsi_semaphore_XXX` calls should just be replaced by the CMSIS OS call as you do for 917 no real reason to have both. It becomes more confusing for the same behaviour.

Reply:
thanks for the comments @jmartinez-silabs. CMSIS OS calls are used instead of `rsi_semaphore_XXX` calls.

Reply:
Done